src/emulator/downloadableEmulators.ts

```diff
@@ -45,9 +45,9 @@ const EMULATOR_UPDATE_DETAILS: { [s in DownloadableEmulators]: EmulatorUpdateDet
   ui: experiments.isEnabled("emulatoruisnapshot")
     ? { version: "SNAPSHOT", expectedSize: -1, expectedChecksum: "" }
     : {
-        version: "1.11.5",
+        version: "1.11.6",
         expectedSize: 3063444,
```

Comment:
This is an odd coincidence but I checked it twice .. thee bytes are the same

Reply:
LOL, thanks for double checking this, i would have been concerned